client/state/sharing/schema.js

```diff
@@ -0,0 +1,19 @@
+import {
```

Comment:
I backed this out in ba8b593, but we still need a way of throwing out `connectionsBySiteId` and `connectionsSchema` if one of them doesn't pass schema checks